### PR TITLE
Assemble about page.

### PR DESCRIPTION
**Pull request by iceicery**
In this PR, I put some component together and add margin to fit the design. However, I was unable to fix the TimeLine issue I found while assembling the page. Please take a look.

### CODE REVIEW

File: src/pages/about.js [hunk 1]
```diff
@@ -2,18 +2,26 @@ import * as React from 'react';
...
+export default function AboutPage({ onClick }) {
   return (
     <Layout>
       <Cookies />
       <Header
         title="About Us"
         text="Data Science = Highly Iterative Metrics-driven Process With Data and Code"
-      />
+      >
+        <TimeLine times={times} />
```
Comment (julieg18) [inline src/pages/about.js:20]: To correct timeline error, change Header.css line 63 from `.header .title {` to `.header .title_size_l {`

File: src/components/Team/Team.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .team {
-  margin: 0 0 40px;
+  margin: 40px 0 40px;
```
Comment (julieg18) [inline src/components/Team/Team.css:2]: `margin: 38px 0 40px is more accurate`

File: src/components/Team/Team.css [hunk 3]
```diff
@@ -53,7 +53,7 @@
 
 @media (min-width: 768px) {
   .team {
-    margin: 0 0 40px;
+    margin: 40px 0 40px;
```
Comment (julieg18) [inline src/components/Team/Team.css:56]: `margin: 60px 0 60px` appears to be more accurate.

File: src/components/Leadership/Leadership.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.leadership {
+  margin: -16px 0 0;
+}
+
+@media (min-width: 1024px) {
+  .leadership {
+    margin: 46px 0 0;
```
Comment (julieg18) [inline src/components/Leadership/Leadership.css:7]: `margin: 0` appears to most accurate.

**Review by julieg18 — CHANGES_REQUESTED** [review]
Awesome job putting everything together! Message me if you need any help!

File: src/components/Header/Header.css [hunk 5]
```diff
@@ -60,7 +60,7 @@
     background-size: 600px;
   }
 
-  .header .title {
+  .header .title_size_l {
```
Comment (iceicery) [inline src/components/Header/Header.css:63]: Interesting. So, how the issue happened? I don't get it.

File: src/pages/about.js [hunk 6]
```diff
@@ -7,12 +7,43 @@ import Footer from '../components/Footer/Footer';
 import Team from '../components/Team/Team';
 import LeaderShip from '../components/Leadership/Leadership';
 import Career from '../components/Career/Career';
+import RequestDemoForm from '../components/RequestDemoForm/RequestDemoForm';
 import { times } from '../utils/data';
 
-export default function AboutPage({ onClick }) {
```
Comment (iceicery) [inline src/pages/about.js]: Ops. I totally forgot we are using gastby. I was in the OLD REACT mind. Good job adding these functions. The form acts perfectly.

**Review by iceicery — COMMENTED** [review]
Thanks for taking care of this. The page looks great and functional.  Nice job!

**Review by julieg18 — APPROVED** [review]

**Pull request merged**